### DISCUSSION TITLE
Any https on the horizon?

### DISCUSSION THREAD

**Discussion by craftdax** [mealie-recipes/mealie#623] [Ideas]
Hey Guys, just started using mealie and I'm loving it! You guys have made something amazing!

I'm someone that also really likes sharing and brainstorming over food and recipes with friends and family so I'm hosting this in my DMZ. I couldn't find any reference to making this work over HTTPS (because I have a voice in my head thats paranoid for not using HTTPS on a page in my DMZ). I was just wondering if you guys had it anywhere on your roadmap? :)

**Comment by hay-kot** [comment]
I don't plan on adding it. I see it as a deployment concern. 

You are correct that you should use HTTPS. In most cases this is done through a reverse proxy. Internally we use Caddy to proxy requests to the API or the static files. If you don't want to use an HTTP proxy you could easily edit the Caddy file an use a docker-mount to overwrite it and enable HTTPS. It will even handle proper certificates.

**Reply by craftdax** [reply]
Ok thanks for the tip, this isn't really my expertise so sorry for the question :)

**Reply by hay-kot** [reply]
No need to apologize, asking questions is how we all learn! 🎉

**Comment by zierbeek** [comment]
I use swag which uses certificates from let's encrypt

**Comment by edward3h** [comment]
I'm proxying with Traefik 2 which also helps manage Let's Encrypt certificates. You can see my setup at https://github.com/edward3h/home_docker_stuff - look in the `core` and `mealie` directories.

**Comment by strenkml** [comment]
I use [Nginx Proxy Manager](https://nginxproxymanager.com/) which uses certificates from Let's Encrypt.  It has a really easy interface and there is no messing around with config files.

**Comment by vdrover** [comment]
@strenkml Are you still using mealie with npm? Anything special setting up the mealie host in npm? not working for me.
